sabelotodo/models.py

```diff
@@ -84,13 +87,19 @@ class Item(db.Model):
     due_date: datetime = db.Column(db.DateTime)
     parent_id: int = db.Column(db.Integer, db.ForeignKey('items.id'))
 
+    def __init__(self, **kwargs):
+        errors = ItemSchema().validate(kwargs)
```

Comment:
I would run `super().__init__()` so that your dataclass functions as you expect it to, and then run your validation.